app/controllers/application_controller.rb

```diff
@@ -49,6 +49,14 @@ def redirect_to_signin
     redirect_to sign_in_path, alert: t("please_sign_in")
   end
 
+  def redirect_to_mfa_setup(**options)
```

Comment:
Do we know what these options will be?

Reply:
Right now the only option used is `notice` to create a flash notice (banner with the warning message). I would imagine that `error` would be another option used to create a flash error (banner with the error message requiring the user to enable MFA) in the future.

https://api.rubyonrails.org/classes/ActionController/Redirecting.html gives the list of options that can be used.